common/scripted_triggers/00_building_requirement_triggers.txt

```diff
@@ -93,7 +93,7 @@ building_pastures_requirement_terrain = {
 		terrain = floodplains
 		AND = {
 			terrain = jungle
-			NOT = { geographical_region = world_innovation_elephants }
+			NOT = { geographical_region = world_innovation_elekk }
```

Comment:
Keep the vanilla name so we don't have to rename `world_innovation_elephants` every patch.
```suggestion
			NOT = { geographical_region = world_innovation_elephants }
```